tests/integration/conftest.py

```diff
@@ -14,7 +14,7 @@ def better_traces(forked_env):
     boa.from_etherscan(ab.vault_original, "vault_original")
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="session")
```

Comment:
Reminder to restrict the scope of this as discussed offline

Reply:
done